src/shared/sam/debugger/awsSamDebugger.ts

```diff
@@ -451,6 +458,7 @@ export class SamDebugConfigProvider implements vscode.DebugConfigurationProvider
             },
             awsCredentials: awsCredentials,
             parameterOverrides: parameterOverrideArr,
+            useIkpdb: isCloud9() || !!(config as any).useIkpdb,
```

Comment:
Swap the order of these arguments? Assuming `useIkpdb` should be prioritized?

Reply:
Only matters in tests

Reply:
The order does not logically matter.